### ISSUE TITLE
Cannot find miner.start/stop in RPC

### ISSUE THREAD

**Issue by nolash** [trufflesuite/ganache-cli#698]
* I asked about this on [gitter](https://gitter.im/ConsenSys/truffle): [ ]
* Link to my question on gitter: 

## Current Behavior

The methods cannot be found. When connecting with `truffle console` and `geth attach` (both http), no `miner` module is listed as available, and I cannot find any mentions of the methods in the `web3` object tree either.

## Possible Solution

If they are indeed available, or if an (undocumented) feature flag has to be passed, please add to documentation.

## Steps to Reproduce (for bugs)

1.
2.
3.
4.

## Your Environment
* Version used:
* Version of Truffle/Remix/Other tools used:
* NodeJS Version: [ ] 6.x, [ ] 7.x (unsupported), [ ] 8.x, [ ] 9.x, [x] 12.x
* Operating System and version (include distro if Linux): linux 5.3.8
* Link to your project or repro gist:
* Commit hash to use with above link for reproduction:
* I intend to submit a pull request to fix this issue: [ ]

**Comment by davidmurdoch** [comment]
It sounds like you are connecting to geth (not ganache-cli)?

Geth's documentation for the management APIs is located here: https://github.com/ethereum/go-ethereum/wiki/Management-APIs#miner_start

**Comment by nolash** [comment]
@davidmurdoch no, I tried to connect to `ganache` with `truffle console` and `geth attach` as clients respectively.

**Comment by davidmurdoch** [comment]
Ok. `miner_start` and `miner_stop` are supported by a ganache. I don't know if web3 has support for these miner management methods though (from your description it sounds like it does not).

Ganache's supported management APIs are documented here: https://github.com/trufflesuite/ganache-cli#management-api-methods
